src/shell.js

```diff
@@ -71,6 +71,24 @@ if (!ENVIRONMENT_IS_PTHREAD) PthreadWorkerInit = {};
 var currentScriptUrl = ENVIRONMENT_IS_WORKER ? undefined : document.currentScript.src;
 #endif
 
+#if EMTERPRETIFY_ASYNC
+Module['notifyAbout'] = function(identifier) {
```

Comment:
Hmm... as long as the function can still be called from JavaScript code as usual, I think this would be perfectly acceptable.